### PR TITLE
Upgrade activesupport to 6.1

### PR DESCRIPTION
**Pull request by cosmo0920**
Note that ActiveSupport 6.1 drops Ruby 2.4 support.
I got CI error response from Travis CI but it is intended failure.

### CODE REVIEW

**Comment by cosmo0920** [comment]
I also added ruby-head task definitions for GitHub Actions.

**Comment by cosmo0920** [comment]
@kenhys @ashie Could you kindly take a look?

File: .github/workflows/linux.yml [hunk 1]
```diff
@@ -9,10 +9,32 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [ '2.4', '2.5', '2.6', '2.7' ]
+        ruby: [ '2.5', '2.6', '2.7', '3.0' ]
         postgres: [ '9', '10', '11', '12', '13' ]
         os:
           - ubuntu-latest
+        experimental: [false]
```
Comment (ashie) [inline .github/workflows/linux.yml:16]: Could you add `continue-on-error: ${{ matrix.experimental }}`?
Otherwise it doesn't take effect.

**Review by ashie — CHANGES_REQUESTED** [review]

File: .github/workflows/linux.yml (hunk 1 above)
Comment (cosmo0920) [reply .github/workflows/linux.yml:17]: Fixed in https://github.com/fluent/fluent-plugin-sql/pull/101/commits/7496d87dbce69a8cc644bd767741107ee751a292.

File: fluent-plugin-sql.gemspec [hunk 2]
```diff
@@ -17,8 +17,8 @@ Gem::Specification.new do |gem|
   gem.license = "Apache-2.0"
 
   gem.add_dependency "fluentd", [">= 0.12.17", "< 2"]
-  gem.add_dependency 'activerecord', "~> 5.1"
-  gem.add_dependency 'activerecord-import', "~> 0.7"
+  gem.add_dependency 'activerecord', "~> 6.1"
```
Comment (ashie) [inline fluent-plugin-sql.gemspec:20]: Memo: It drops older fluentd due to tzinfo version conflict issue, but I think it's O.K.

**Review by ashie — APPROVED** [review]

**Comment by cosmo0920** [comment]
Thanks! :rocket:

**Comment by ashie** [comment]
I'll release the next version.

File: fluent-plugin-sql.gemspec [hunk 3]
```diff
@@ -17,8 +17,8 @@ Gem::Specification.new do |gem|
   gem.license = "Apache-2.0"
 
   gem.add_dependency "fluentd", [">= 0.12.17", "< 2"]
```
Comment (ashie) [inline fluent-plugin-sql.gemspec:19]: Hmm, we should also update required fluend version.

**Pull request merged**